tests/integration/cairo1_compiler_bindings/library_functions_test/prepare_test.cairo

```diff
@@ -5,13 +5,19 @@ fn test_prepare() {
    let mut arr = ArrayTrait::new();
    arr.append(101);
    arr.append(202);
-   arr.append(303);
-   arr.append(405);
-   arr.append(508);
    arr.append(613);
    arr.append(721);
    match prepare_tp(123, arr) {
```

Comment:
We should use prepare instead of prepare_tp here. Will address it in separate PR and create issue.

Reply:
https://github.com/software-mansion/protostar/pull/1705